docs/testing.md

```diff
@@ -17,6 +17,10 @@ If the tests are failing and you want to see the difference between the image yo
 ## Test docker image
 
 The docker image has tests that you can run on it.
-You need Docker, bash, and bats available on your system.
 
-Once you've got a built image, run `./tests/bats/test.sh custom-grafana-image-renderer`.
+`IMAGE=X go test ./tests/acceptance/...`
+
+The image will be retrieve from docker hub if not present locally (might require login).
```

Comment:
nope, you need to pull it yourself first :)

Reply:
gotcha, misinterpreted the error message I was getting then :)